modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -21,21 +21,35 @@ The following table lists the temporal value types and their supported component
 [options="header", cols="^,^,^,^"]
 |===
 | Type            | Date support | Time support | Time zone support
-| `Date`          | {check-mark} |              |
-| `Time`          |              | {check-mark} | {check-mark}
-| `LocalTime`     |              | {check-mark} |
-| `DateTime`      | {check-mark} | {check-mark} | {check-mark}
-| `LocalDateTime` | {check-mark} | {check-mark} |
-| `Duration`      | `-`          | `-`          | `-`
+| `DATE`          | {check-mark} |              |
+| `LOCAL TIME`     |              | {check-mark} |
+| `ZONED TIME`          |              | {check-mark} | {check-mark}
+| `LOCAL DATETIME` | {check-mark} | {check-mark} |
+| `ZONED DATETIME`      | {check-mark} | {check-mark} | {check-mark}
+| `DURATION`      | `-`          | `-`          | `-`
 |===
 
-`Date`, `Time`, `LocalTime`, `DateTime`, and `LocalDateTime` are _temporal instant_ types.
+`DATE`, `LOCAL TIME`, `ZONED TIME`, `LOCAL DATETIME`, and `ZONED DATETIME` are _temporal instant_ types.
 A temporal instant value expresses a point in time with varying degrees of precision.
 
-By contrast, `Duration` is not a temporal instant type.
-A `Duration` represents a temporal amount, capturing the difference in time between two instants, and can be negative.
-`Duration` captures the amount of time between two instants, it does not capture a start time and end time.
+By contrast, `DURATION` is not a temporal instant type.
+A `DURATION` represents a temporal amount, capturing the difference in time between two instants, and can be negative.
+`DURATION` captures the amount of time between two instants, it does not capture a start time and end time.
 
+Some temporal types have been renamed slightly.
+The table below shows the current as well as the old names of the temporal types.
+_This change was introduced in Neo4j 5.9._
```

Comment:
`slightly` is irrelevant - the amount of implementation work to comply with new names is the same regardless of the size of the renaming. I'd remove that.

I'd actually group the first and last sentence as "Starting from Neo4j 5.9, some temporal types have been renamed."